scripts/run_integration_tests.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+# Usage - run commands from repo root:
+# To check if new changes to the plugin cause changes to any snapshots:
+#   ./scripts/run_integration_tests
+# To regenerate snapshots:
+#   UPDATE_SNAPSHOTS=true ./scripts/run_integration_tests
+
+set -e
+
+script_path=${BASH_SOURCE[0]}
+scripts_dir=$(dirname $script_path)
+repo_dir=$(dirname $scripts_dir)
+root_dir=$(pwd)
+#/Users/andrew.rodriguez/go/src/github.com/DataDog/serverless-plugin-datadog
+if [[ "$root_dir" =~ .*"serverless-plugin-datadog/scripts".* ]]; then
+    echo "Make sure to run this script from the root `serverless-plugin-datadog` directory, aborting"
+    exit 1
+fi
+
+integration_tests_dir="$repo_dir/integration_tests"
+
+if [ -n "$UPDATE_SNAPSHOTS" ]; then
+    echo "Overwriting snapshots in this execution"
+fi
+
+cd $integration_tests_dir
+yarn
+
+# Add local build to node_modules so `serverless-forwarder.yml` and `serverless-extension.yml` also have access to local build.
+cd $root_dir
+yarn
+yarn build
+rm -rf "$integration_tests/node_modules"
+mkdir -p "$integration_tests_dir/node_modules/serverless-plugin-datadog"
```

Comment:
So, a trick you can do with the serverless plugins, is to put them into the [serverless plugins](https://www.serverless.com/framework/docs/providers/aws/guide/plugins#service-local-plugin) directory, instead of node_modules. Or altenatively, you can might be able to change 'serverless-plugin-datadog' to a local path, eg '../dist'.

Reply:
Yup I removed the logic of adding the local plugin to the node modules and instead just referenced the local plugin in the yml files.